### PR TITLE
[INTLY-5109] Create AWS Resource Alerts

### PR DESCRIPTION
**Pull request by dimitraz**
### Description 
Jira: https://issues.redhat.com/browse/INTLY-5109

### Verification
 - Run an install of integreatly from this branch
- Ensure `useClusterStorage` is set to false 
- Wait for all RDS and Elasticache instances to come up 
- Navigate to the prometheus console, ensure all relevant alerts are green

### CODE REVIEW

File: deploy/test/test-idp-templace.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      source: ${OPERATOR_SOURCE_NAME}
+parameters:
+  - description: The namespace to deploy into
+    displayName: Namespace
+    name: NAMESPACE
+    value: redhat-rhmi-operator
+  - description: Password used for testing users
+    displayName: Password used for testing users
+    name: PASSWORD
+    value: Password1
```
Comment (mikenairn) [inline deploy/test/test-idp-templace.yml]: Think this is meant to template deploy/test/test-idp-templace.yml

**Comment by david-martin** [comment]
@dimitraz Failing test shows below alerts triggering

```
            integreatly_test.go:691: alert tests failed: [Firing alerts: [3scaleRedisCacheUnavailable codeready-workspacesPostgresInstanceUnavailable 3scaleRedisCacheUnavailable 3scalePostgresInstanceUnavailable upsPostgresInstanceUnavailable] Pending alerts: [SolutionExplorerPodCount]]
```

I'm unsure about the Pending alert for SolutionExplorerPodCount,
but definitely worth looking into why the newly added alerts are failing.
Do e2e tests run in an aws environment and does CRO provision with an aws strategry (not in cluster) for the e2e tests?

**Comment by dimitraz** [comment]
@david-martin Completely my fault, I forgot to test this with in-cluster resources. The e2e tests are run with the `useClusterStorage` flag set to `true`, so those alerts are fired because I didn't handle the Openshift side of things in CRO. For now I'll wrap the alerts in a check to ensure they aren't created when that flag is set. 

If you think it's worthwhile to spend time on I can implement the Openshift metrics tomorrow

**Pull request merged**